### ISSUE TITLE
[undefined] before Request Body and Response Body

### ISSUE THREAD

**Issue by unkownymouse** [sirrodgepodge/morgan-body#24]
When the request and response is printed on the console, there is a `[undefined] Request Body:` instead of `Request Body:` as shown in ScreenShots.

<img width="695" alt="Screenshot 2020-02-12 at 6 49 20 PM" src="https://user-images.githubusercontent.com/32857628/74338781-d45ec500-4dc8-11ea-8b8c-a343dda41a39.png">

The minimal server to reproduce:
```javascript
const express = require('express');
const morgan = require('morgan-body');

const app = express();
app.use(express.json());
morgan(app);

app.get('/', function (req, res) {
    console.log(req.body);
    res.send(req.body);
    res.end()
});


app.listen(4001);
```

Any clue about why its displaying `[undefined] Request Body:` instead of `Request Body:`?

**Comment by sirrodgepodge** [comment]
ah great catch, thank you!  I'm guessing this is related to an API change in the latest version of Express, will check it out

**Comment by starpeng** [comment]
The [undefined] is req.id
We can fix it by adding `express-request-id` middleware

```
const express = require('express')
const morgan = require('morgan-body')
const expressRequestId = require('express-request-id')

const app = express()
app.use(expressRequestId())
app.use(express.json())
morgan(app)

app.get('/', function (req, res) {
    console.log(req.body)
    res.send(req.body)
    res.end()
});


app.listen(4001)
```

**Comment by johnantoni** [comment]
nice find @starpeng that fixed it

**Comment by mohammadzafardoost** [comment]
Hey @sirrodgepodge Thanks for this good package,
Please fix this problem sooner so we don't have to use 'express-request-id'

**Comment by Passiday** [comment]
The problem lies in the line #320 of `index.js`. The `getIDToken(req)` can return `undefined`, if the `req.id` is not set. Whilte the Express` original logger handles that gracefully by outputting "[-]", your code does not check for that `undefined` value. Here's the fix:

    const IDToken = getIDToken(req);
    const exPrependStr = '[' + (IDToken ? IDToken : '-') + '] ' + prependStr;

**Comment by sirrodgepodge** [comment]
fixed thanks to @Passiday and @olvis

**Comment by sirrodgepodge** [comment]
fixed in version 2.4.9
